pepper-apis/src/main/java/org/broadinstitute/ddp/model/copy/CopyConfiguration.java

```diff
@@ -0,0 +1,43 @@
+package org.broadinstitute.ddp.model.copy;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.jdbi.v3.core.mapper.reflect.ColumnName;
+import org.jdbi.v3.core.mapper.reflect.JdbiConstructor;
+
+public class CopyConfiguration {
+
+    private long id;
+    private long studyId;
+    private List<CopyConfigurationPair> pairs = new ArrayList<>();
+
+    @JdbiConstructor
```

Comment:
Another dumb compiler warning.
```suggestion
    @SuppressWarnings("unused")
    @JdbiConstructor
```